addons/interact_menu/initSettings.sqf

```diff
@@ -7,3 +7,57 @@
     false,
     {GVAR(selectorColorHex) = _this call BIS_fnc_colorRGBtoHTML} // Stored in Hex to avoid constant conversion
 ] call CBA_settings_fnc_init;
+
+[
+    QGVAR(alwaysUseCursorInteraction),
+    "CHECKBOX",
+    localize LSTRING(AlwaysUseCursorInteraction),
```

Comment:
`LLSTRING` and `LELSTRING` could be used here and below
https://github.com/CBATeam/CBA_A3/blob/6a818a7c9780ce316cfcfad6fbeff199e6c56666/addons/main/script_macros_common.hpp#L1198-L1199